### ISSUE TITLE
Please clarify historical relations to hyperHTML?

### ISSUE THREAD

**Issue by AndyOGo** [Polymer/lit-html#903]
This weekend I was a bit shocked to be honest, after I read **The history of hyperHTML followed by lit-html** by @WebReflection:
https://gist.github.com/WebReflection/ab43649d9e4a53ac900b5924c77a310e

I'm really surprised and wonder if this projected commits plagiarism of [`hyperHTML`](https://github.com/WebReflection/hyperHTML).
Please clarify the situation.

**Comment by WebReflection** [comment]
Hi @AndyOGo, I can clarify the situation for you, since I believe nobody in here would commit "_seppuku_" anyway.

  * lit-html didn't commit plagiarism, it likely came out of very similar developers idea to use template literals tags
  * lit-html went out with different concepts, where few of them are weaker, but few other are stronger, so that overall it's not a 1:1 drop-in replacement for hyperHTML
  * I wrote that gist to underline that instead of helping an _incredibly similar_ Open Source project and its developer, at that time also trying to survive out of Patreon or OpenCollective, this team didn't bother at all, and went ahead full steam, even rarely mentioning the project that predated its own library concepts at least 6 months before

TL;DR That gist is about a sad story regarding modern Open Source dynamics, but it doesn't want to claim plagiarism from Google, it simply wants to underline how similar were the ideas, and how this team avoided any sort of reach/cooperation whatsoever.

**Comment by AndyOGo** [comment]
Thanks for your real quick clarification @WebReflection .
Honestly I was quite irritated, all fine now.

**Comment by justinfagnani** [comment]
@AndyOGo Personally, I'd love to comment more but professionally I can't.
